src/mobu/handlers/external.py

```diff
@@ -15,9 +16,22 @@
     MonkeyBusinessManager,
     monkey_business_manager,
 )
-from ..models import Index
-
-__all__ = ["external_router", "get_index", "get_users", "post_user"]
+from ..models.error import ErrorModel
```

Comment:
You might want a convenience method to get logs for all monkeys in a flock without having to manually walk the list...but maybe not.